OpenCover.UI.TestDiscoverer.TestResources/Xunit/RegularTestFixture.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Xunit;
+
+namespace OpenCover.UI.TestDiscoverer.TestResources.Xunit
+{    
+	public class RegularFacts
+	{
+		[Fact]
```

Comment:
@joshlrogers Actually my last comment maybe misleading or pointing out the issue in the wrong spot.. This is probably why the build is still failing, as the test discoverer sees this as a real xunit test and hence treating `OpenCover.UI.TestDiscoverer.exe` as a test container..